manifests/configfile.pp

```diff
@@ -40,8 +40,8 @@
   include logstash
 
   $path = "${logstash::config_dir}/conf.d/${name}"
-  $owner = $logstash::logstash_user
-  $group = $logstash::logstash_group
+  $owner = 'root'
+  $group = 'root'
   $mode ='0440'
```

Comment:
Looks like the config files won't be readable by the unprivileged user.

We could either make them `444 root:root`, or (probably better) `440 root:$logstash:logstash_group`.